.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!-- Hello! Thanks for submitting a PR. To help make things go a bit more
+     smoothly we would appreciate that you go through this template.
+-->
+
+### Description
+
+<!-- Good things to put here include: reasoning for the change (please link
+     any relevant issues!), any noteworthy (or hacky) choices to be aware of,
+     or what the problem resolved here looked like ... we won't mind a ranty
+     story :)
+-->
+
+### Checklist - did you ...
+
+<!-- If any of the following items aren't relevant for your contribution
+     please still tick them so we know you've gone through the checklist.
+
+    Tests are required for bugfixes and new features. Documentation changes
+    are necessary for formatting and most enhancement changes.
+
+    Ensure proper code formatting with black. You can do this by running
+    black from landlab's top-level folder.
+
+    Ensure landlab is lint-free by running flake8 at landlab's top-level
+    folder.
+
+    If you like, you can automate these tasks by installing pre-commit hooks.
+    This is done by running `pre-commit install` at landlab's top-level
+    folder.
+-->
+
+- [ ] Add a [news fragment](https://landlab.readthedocs.io/en/master/development/contribution/index.html#news-entries) file entry if necessary?
+- [ ] Add / update tests if necessary?
+- [ ] Add new / update outdated documentation?
+- [ ] All tests have passed?
+- [ ] Formatted code with black?
+- [ ] Removed lint reported by flake8?
```

Comment:
Is there also a way to indicate the thing about the boring title underline rejection because of too short?

Reply:
I think that's a little too specific to include here. How about a checkbox to indicate that, if documentation is being added or modified, you've successfully built to docs locally? We could include a link to instructions about how to build the docs locally.

Reply:
great!